game/src/Levels/SkyParallaxBackground.tscn

```diff
@@ -4,26 +4,26 @@
 [ext_resource path="res://assets/environment/background/cloud_small_1.png" type="Texture" id=2]
 [ext_resource path="res://assets/environment/background/cloud_small_2.png" type="Texture" id=3]
 [ext_resource path="res://assets/environment/background/mountains.png" type="Texture" id=4]
+[ext_resource path="res://assets/environment/background/sky.png" type="Texture" id=5]
 
-[sub_resource type="Gradient" id=1]
-offsets = PoolRealArray( 0, 0.813609 )
-colors = PoolColorArray( 0.184314, 0.564706, 0.831373, 1, 0.364706, 0.705882, 0.843137, 1 )
+[sub_resource type="GDScript" id=1]
+script/source = "extends Sprite
 
-[sub_resource type="GradientTexture" id=2]
-gradient = SubResource( 1 )
-width = 1080
 
-[node name="SkyParallaxBackground" type="ParallaxBackground"]
+func _process(delta: float) -> void:
+	global_position.y = get_viewport_transform().origin.y + get_viewport_rect().size.y
+"
```

Comment:
This feels unnecessary to do every frame. If the goal is to pin the Y position of the mountains in place so they aren't affected by the parallax background, then setting the mountains' parallax background's motion scale on the Y to 0 will do that just fine.

ParallaxBackground3.motion_scale.y = 0
Mountains.position.y = 222 (texture's height/2)

No need for a script on the mountains at all.

Reply:
Did you test this solution?

Reply:
I played around a bit to find it, yeah

Reply:
Did you test this solution? Doesn't seem to work on my end.

Reply:
Ohh, thanks. Yeah this totally solved it.